### DISCUSSION TITLE
Add sort options to library

### DISCUSSION THREAD

**Discussion by kevin122673** [Taiizor/Sucrose#80] [Ideas]
Add Library Sorting
  Sort By:  [Name | Date Added | Modification Date]
  Reverse Order [OFF | ON]
![library-sort-order](https://github.com/Taiizor/Sucrose/assets/103470356/d63c876e-3e1c-4590-8ade-e8990bdcfbed)

**Comment by Taiizor** [comment]
Great idea. I've taken note of this and will start working on its development soon.

**Comment by Taiizor** [comment]
Development of this feature has concluded. This feature will be available in the next Sucrose release.

![Sucrose-Wallpaper-Engine](https://github.com/Taiizor/Sucrose/assets/41683699/5befe5bb-0461-4f81-94d9-630aba00a27a)

https://github.com/Taiizor/Sucrose/assets/41683699/09ab40b5-6c28-4765-af48-d8f1e2bf4ce4
